docs/Doxyfile.in

```diff
@@ -943,7 +943,11 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  =
+INPUT                  = @NOTE_C_BASE@/n_hooks.c \
```

Comment:
We only need these files for the documentation, as it exists now. We don't document every API on the docs site, which is intentional. Only the essential stuff.